justfile

```
@@ -89,7 +90,7 @@ precommit:
 
 # Run pre-commit to lint and reformat files
 lint hook="" *files="": precommit
-    python3 pre-commit.pyz run {{ hook }} {{ if files == "" { "--all-files" } else { "--files" } }}  {{ files }}
+    python3 pre-commit.pyz run {{ hook }} {{ if files =~ "all-files" { "" } else if files == "" { "--all-files" } else { "--files" } }}  {{ files }}
```

Can this be broken down at least into two lines to make it more readable?
```suggestion
    python3 pre-commit.pyz run {{ hook }} \
    {{ if files =~ "all-files" { "" } else if files == "" { "--all-files" } else { "--files" } }}  {{ files }}
```

---

I can remove this change entirely now.